### PR TITLE
Expose RPC as onion service

### PR DESCRIPTION
**Pull request by MarnixCroes**

### CODE REVIEW

File: docs/using-wasabi/RPC.md [hunk 1]
```diff
@@ -648,6 +649,58 @@ Stops and exits Wasabi.
...
+
+```bash
+wassabeed --rpconionenabled=true
+```
+
+The onion address is shown in the log:
+`INFO       Global.StartTorProcessManagerAsync (284)        RPC server listening on http://rrdayxv2pngzl3jyal5dfjvl6s4bt4frvo5jj2rgnajz5gyevrm4fvyd.onion/`
+
+The onion service cannot be started in case the _jsonrpcuser_ and/or _jsonrpcpassword_ are not specified and it will log `Anonymous access RPC server cannot be exposed as onion service.`
+You then need to specify them in the config file or add them as command line switches.
```
Comment (lontivero) [inline docs/using-wasabi/RPC.md:676]: or var envs

**Review by lontivero — DISMISSED** [review]

**Review by MaxHillebrand — APPROVED** [review]

File: docs/using-wasabi/RPC.md [hunk 2]
```diff
@@ -648,6 +649,58 @@ Stops and exits Wasabi.
 curl -s --data-binary '{"jsonrpc":"2.0", "method":"stop"}' http://127.0.0.1:37128/ | jq
 ```
 
+## Expose RPC Server as onion service
+
+Since Wasabi version [2.0.6](https://github.com/zkSNACKs/WalletWasabi/releases/tag/v2.0.6) the RPC can be exposed as an onion service, which enables remote control.
+
+The RPC Server can be exposed as an onion service by using the _rpconionenabled=true_ [start up parameter](/using-wasabi/StartupParameters.md) or environment variable.
```
Comment (yahiheb) [inline docs/using-wasabi/RPC.md:656]:
```suggestion
The RPC Server can be exposed as an onion service by using the _rpconionenabled=true_ [start up parameter](/using-wasabi/StartupParameters.md) or environment variables.
```

File: docs/using-wasabi/RPC.md [hunk 3]
```diff
@@ -648,6 +649,58 @@ Stops and exits Wasabi.
...
+
+A few notes:
+- _Tor_ needs to be enabled
+- Anonymous access is not allowed: _jsonrpcuser_ & _jsonrpcpassword_ need to be specified.
+- _RPCOnionEnabled_ is only available as command line switch (start up parameter) and environment variable, it is not available in the config file.
```
Comment (yahiheb) [inline docs/using-wasabi/RPC.md:661]:
```suggestion
- _RPCOnionEnabled_ is only available as a command line switch (start up parameter) and an environment variable, it is not available in the config file.
```

**Review by yahiheb — CHANGES_REQUESTED** [review]

File: docs/using-wasabi/RPC.md [hunk 4]
```diff
@@ -648,6 +649,58 @@ Stops and exits Wasabi.
...
 ```
 
+## Expose RPC Server as onion service
+
+Since Wasabi version [2.0.6](https://github.com/zkSNACKs/WalletWasabi/releases/tag/v2.0.6) the RPC can be exposed as an onion service, which enables remote control.
+
+The RPC Server can be exposed as an onion service by using the _rpconionenabled=true_ [start up parameter](/using-wasabi/StartupParameters.md) or environment variable.
+
+A few notes:
+- _Tor_ needs to be enabled
```
Comment (yahiheb) [inline docs/using-wasabi/RPC.md:659]:
```suggestion
- _Tor_ needs to be enabled.
```

**Review by yahiheb — COMMENTED** [review]
LGTM

Fix the conflicts to merge this.

**Review by yahiheb — APPROVED** [review]
LGTM

**Pull request merged**